examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -8076,7 +8085,7 @@ endpoint 1 {
     callback attribute acceptedCommandList;
     callback attribute attributeList;
     ram      attribute featureMap default = 0;
-    ram      attribute clusterRevision default = 1;
+    ram      attribute clusterRevision default = 2;
```

Comment:
Recommend moving to AttributeAccessInterface-provided attribute for ClusterRevision in a subsequent PR as it scales better, since the revision associated with the code of the cluster likely is fixed.

Reply:
agreed